java/src/main/java/com/factset/protobuf/stach/extensions/v2/ColumnOrganizedStachDecompress.java

```diff
@@ -0,0 +1,90 @@
+package com.factset.protobuf.stach.extensions.v2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Iterator;
+
+import org.json.JSONObject;
+import org.json.JSONArray;
+
+public class ColumnOrganizedStachDecompress {
```

Comment:
Minor: Shall we rename the class to `ColumnOrganizedStachUtilities`? That way we can reuse the class for any other helper methods that need to be added in the future.

@Ramanth FYI

Reply:
Sure let's keep it consistent with the other packages.